applications/PolkaBot_Lovable_for_Polkadot.md

```diff
@@ -0,0 +1,189 @@
+
+ - Hackathon MVP proved the process of creating Polkadot Dapps is possible.
+ - Current focus is improving reliability of all components, including IDE, LLM prompt chains, and evals.
+ - Next steps include supporting more common Polkadot Dapps and integration work outlined in the Project Details section.
+
+## 📅 Development Roadmap
+
+**Milestone 1:** PolkaBot IDE / Frontend
+ - Create (and understand) the frontend components and UI required to build dapps in the browser.
+ - Grant team/User can open a live app and prompt for code, edit the code editor, prompt for bash commands, and see the rendered website (output).
+ - KPI: Frontend shown to 3 Polkadot developers and feedback gathered for usability
+
+| item | days | rate |
+|------|------|------|
+| Chat UI | 3 | $250 |
+| Website renderer | 2 | $250 |
+| In-browser filestorage | 2 | $250 |
+| Runtime environment | 3 | $250 |
+| Build/compilation tools | 2 | $250 |
+| Integration of the above | 2 | $250 |
+| Usability research and tweaks | 2 | $250 |
+| **Total** | 16 | $4000 |
+
+
+**Milestone 2:** Polkadot app generation supported
+ - Prompts can now generate a working Polkadot app with high reliability, targeting Polkadot Hub
+ - How to demo: Users can generate apps and see that the code interfaces with polkadot hub and assets can be managed on Polkadot Hub.
+ - Grant team/User can enter a prompt and receive a rendered app, with the ability to refine via follow-up prompts. Works well for one app type.
+ - KPI: Product used by 3 developers, and final app / code feedback gathered for usability
+
+| item | days | rate |
+|------|------|------|
+| Learn to build on Polkadot, and how to teach LLMs to do it | 3 | $250 |
+| Example code snippets for LLM to recreate this | 3 | $250 |
+| Optimise planning LLM | 2 | $250 |
+| Optimise agentic systems | 8 | $250 |
+| Build eval sets (testing but for LLMs) | 2 | $250 |
+| Usability research and tweaks | 2 | $250 |
+| **Total** | 20 | $5,000
+
+
+**Milestone 3:** Extensibility and Documentation
+ - Publish a public article and share results with the Polkadot developer community.
+ - Document how to expand the AI’s capabilities to support more project types.
+ - Implement unit testing, refactor codebase, and enable community contribution.
+ - KPI: Deliver comprehensive documentation, testing guide, and notes for future contributors.
+
+| item | days | rate |
+|------|------|------|
+| Article | 1 | $250 |
+| Implement Testing, refactor | 2 | $250 |
+| Documentation on expanding capabilities | 1 | $250 |
+| Total | 4 | $1000 |
+
+
+### Overview
+
+- **Estimated Duration:** 3mo
+- **Full-Time Equivalent (FTE):**  1.5 FTE
+- **Total Costs:** $10,000
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License: MIT |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can contribute to the open source project |
```

Comment:
Do you mean a tutorial that explains how to run the tool ? As a reviewer it would be nice that you include clear instructions about how to test from an end user's perspective in addition to what you outlines in 0c.